app/views/admin/debate_outcomes/show.html.erb

```diff
@@ -29,6 +29,6 @@
   <% end %>
 
   <%= form_row do %>
-    <%= f.submit 'Publish & Send Email', data: { confirm: "Are you sure you want to publish these debate outcome details and email all of the supporters?" } %>
+    <%= f.submit "Email #{number_with_delimiter(@petition.signature_count)} #{'signature'.pluralize}" %>
```

Comment:
It might be good to extract this pluralizing to a helper, the pattern is used in at least 4 places now.